packages/emails/src/components/LocationInfo.tsx

```diff
@@ -24,8 +24,7 @@ export function LocationInfo(props: { calEvent: CalendarEvent; t: TFunction }) {
   // Because of location being a value here, we can determine the app that generated the location only for Dynamic Link based apps where the value is integrations:*
   // For static link based location apps, the value is that URL itself. So, it is not straightforward to determine the app that generated the location.
   // If we know the App we can always provide the name of the app like we do it for Google Hangout/Google Meet
-
-  if (meetingUrl) {
+  if (meetingUrl && !props.calEvent.requiresConfirmation) {
```

Comment:
Ideally `meetingUrl` shouldn't be there in calEvent if booking requires confirmation. 

This is the reason we can't have a meeting link for such a booking.

If we fix this, we won't need to use `requiresConfirmation` here.

Reply:
I think it's okay and make sense to add the `requireconfirmation` check. What you think?

Reply:
I think it doesn't. You can't really have a meeting URL for a booking that isn't confirmed. So, we might add this check here and fix the email content but the system still has meeting link for unconfirmed booking which we can accidentally show up elsewhere.

Reply:
I will fix that

Reply:
Hey Hariom, when generating or setting a meeting URL, we need to extract values from several fields, especially for dynamic link apps, not just a single field of calEvent. Additionally, if we don't set calEvent location-related details at the time of booking, how do we trace or extract those values after confirming the booking?

Reply:
cc @hariombalhara